src/stuf/stuf_zds/templates/stuf_zds/soap/creeerZaak.xml

```diff
@@ -57,16 +57,11 @@
         {% if initiator.bsn or initiator.kvk %}
             <ZKN:heeftAlsInitiator StUF:verwerkingssoort="T" StUF:entiteittype="ZAKBTRINI">
                 <ZKN:gerelateerde>
-                    {% comment SUPPORT FOR EMPLOYEES AS INITIATOR %}
-                    {% if initiator.TODO %}
+                    {% if medewerker %}
                         <ZKN:medewerker StUF:entiteittype="MDW" StUF:verwerkingssoort="T">
-                            <ZKN:identificatie>?</ZKN:identificatie>
-                            <ZKN:achternaam>?</ZKN:achternaam>
-                            <ZKN:voorletters>?</ZKN:voorletters>
-                            <ZKN:voorvoegselAchternaam>?</ZKN:voorvoegselAchternaam>
+                            <ZKN:identificatie>{{ medewerker.identificatie }}</ZKN:identificatie>
```

Comment:
this is wrong and the previous version needs to restored - there's situations (for internal forms) where the employee _is_ the initiator, but that's not for this PR.

Instead, I expect this structure (from the Zaak & Documentservices XSD):

```xml
            <ZKN:heeftAlsOverigBetrokkene StUF-verwerkingssoort="T" StUF:entiteittype="ZAKBTROVR">
                <ZKN:gerelateerde>
                    <ZKN:medewerker StUF:entiteittype="MDW" StUF:verwerkingssoort="T">
                        <ZKN:identificatie>...</ZKN:identificatie>
                        <!-- optional
                        <ZKN:achternaam>?</ZKN:achternaam>
                        <ZKN:voorletters>?</ZKN:voorletters>
                        <ZKN:voorvoegselAchternaam>?</ZKN:voorvoegselAchternaam>
                        -->
                    </ZKN:medewerker>
                </ZKN:gerelateerde>
                <ZKN:toelichting>...</ZKN:toelichting>
            </ZKN:heeftAlsOverigBetrokkene>
```

This goes on the same level as `<ZKN:heeftAlsInitiator>`